features/http_cache/tags.feature

```diff
@@ -17,15 +17,15 @@ Feature: Cache invalidation through HTTP Cache tags
     }
     """
     Then the response status code should be 201
-    And the header "Cache-Tags" should not exist
+    And the header "Cache-Tag" should not exist
```

Comment:
It doesn't break the old code because the old code didn't work anymore since mid 2015.

Reply:
You may implement this in a VCL manually and yes it is working .

Reply:
As it's configurable from the API Platform configuration, feel free to set the Cache-Tags header instead of the Cache-Tag one. Cloudflare doesn't support Cache-Tags (the one **with** a trailing s) header anymore and only the Cache-Tag (the one **without** the trailing slash).